base/distro.bzl

```diff
@@ -1 +1,5 @@
 DISTROS = ["debian10", "debian11"]
+
+# language images (java, node, python, etc) have shorter support windows
```

Comment:
maybe "RUNTIME_DISTROS"? To be more descriptive as to what has a shorter support horizon

Reply:
yeah, unfortunately we still have CC support for a bit longer (I think kubernetes uses it). I'm okay renaming it, but I don't know if `RUNTIME_DISTROS` is any more descriptive? But I also don't have strong opinions on this. Kind of a problem of having all these combinations of config.

Reply:
fair enough!